core/sql/executor/ExStats.cpp

```diff
@@ -2741,13 +2749,21 @@ Int64 ExHbaseAccessStats::getNumVal(Int32 i) const
 
 NABoolean ExHbaseAccessStats::filterForSEstats(struct timespec currTimespec, Lng32 filter)
 {
-   blockTime_ = timer_.filterForSEstats(currTimespec);
-   if (blockTime_ >= filter)
-      return TRUE;
+   Int64 sumIOTime;
```

Comment:
Yes. Can be done. I am thinking of getting rid of one of these classes because both have same member variables sometime in future.